test/custom/BidWhenPaused.test.js

```diff
@@ -8,7 +8,7 @@ const { eth, bidThroughParty, contribute } = require('../helpers/utils');
 const { placeBid } = require('../helpers/externalTransactions');
 const { deployTestContractSetup } = require('../helpers/deploy');
 const { MARKETS, MARKET_NAMES } = require('../helpers/constants');
-const { testCases } = require('../partyBidTestCases.json');
+const { testCases } = require('../partybid/partyBidTestCases.json');
 
 describe('Bid When Paused', async () => {
```

Comment:
thoughts on the casing of the folders? `PartyBid` vs. `partybid`? https://github.com/PartyDAO/partybid/pull/69#discussion_r783501589